### PR TITLE
Hunting queries windows event batch1

### PR DESCRIPTION
**Pull request by yaelrbergman**
Required items, please complete
   
  Change(s):
Add Windows Event Data to the following Hunting Queries: 

Hunting Queries/MultipleDataSources/Dev-0056CommandLineActivityNovember2021.yaml
Hunting   Queries/MultipleDataSources/NICKELCommandLineActivity-Nov2021.yaml
Hunting Queries/MultipleDataSources/NickelRegIOCPatterns.yaml
Hunting   Queries/MultipleDataSources/PersistViaIFEORegistryKey.yaml
Hunting   Queries/MultipleDataSources/PotentialMicrosoftSecurityServicesTampering.yaml
Hunting   Queries/MultipleDataSources/SolarWindsInventory.yaml
Hunting   Queries/MultipleDataSources/TrackingPasswordChanges.yaml

   Reason for Change(s):
part of adding Windows Event data source project

   Version Updated:
No
   Testing Completed:
Yes
   Checked that the validations are passing and have addressed any issues that are present:
Yes

### CODE REVIEW

File: Hunting Queries/MultipleDataSources/Dev-0056CommandLineActivityNovember2021.yaml [hunk 1]
```diff
@@ -37,6 +37,17 @@ query: |
...
     | project TimeGenerated, Computer, NewProcessName, ParentProcessName, Account, NewProcessId, Type, CommandLine
     | extend timestamp = TimeGenerated, HostCustomEntity = Computer , AccountCustomEntity = Account, ProcessCustomEntity = NewProcessName
+    ),
+    (WindowsEvent
+    | where EventID == '4688' and (EventData has_any("/q /c color f7&", "Net.We$()bClient", "$b,15,$b.Length-15") or (EventData has "FromBase64String" and EventData has_all("-nop", "iex", "(iex")) )
+    | extend CommandLine = tostring(EventData.CommandLine) 
+    | where ( CommandLine has_any("/q /c color f7&", "Net.We$()bClient", "$b,15,$b.Length-15")) or (CommandLine has "FromBase64String" and CommandLine has_all("-nop", "iex", "(iex"))
+    | extend NewProcessName = tostring(EventData.NewProcessName)
+    | extend ParentProcessName = tostring(EventData.ParentProcessName)
+    | extend Account =  strcat(tostring(EventData.SubjectDomainName),"\\", tostring(EventData.SubjectUserName))
```
Comment (oshezaf) [inline Hunting Queries/MultipleDataSources/Dev-0056CommandLineActivityNovember2021.yaml:47]: Shouldn't we check for an empty domain (and probably also "-") and if so not add "\\"?

File: Hunting Queries/MultipleDataSources/PotentialMicrosoftSecurityServicesTampering.yaml [hunk 2]
```diff
@@ -46,6 +49,31 @@ query: |
   | project TimeGenerated, Computer, Account, AccountDomain, ProcessName, ProcessNameFullPath = NewProcessName, EventID, Activity, CommandLine, EventSourceName, Type
   ),
   (
+  WindowsEvent
+  | where EventID == 4688 
+  | where (EventData  has_any (action) and EventData has_any (service1))
```
Comment (oshezaf) [inline Hunting Queries/MultipleDataSources/PotentialMicrosoftSecurityServicesTampering.yaml:54]: - filter for has_any(includeProc) prior to the big and complex where clause driven from the CommandLine filter.

**Review by oshezaf — CHANGES_REQUESTED** [review]

File: Hunting Queries/MultipleDataSources/Dev-0056CommandLineActivityNovember2021.yaml (hunk 1 above)
Comment (yaelrbergman) [reply Hunting Queries/MultipleDataSources/Dev-0056CommandLineActivityNovember2021.yaml:47]: @oshezaf Done. Should I add it everywhere else?

Comment (oshezaf) [reply Hunting Queries/MultipleDataSources/Dev-0056CommandLineActivityNovember2021.yaml]: I think so, but maybe phase 2, as it is a lot of work.

**Comment by vmanojreddy** [comment]
@aprakash13  Please review hunting queries. thanks

**Review by oshezaf — APPROVED** [review]

**Pull request merged**